.github/ISSUE_TEMPLATE/1-bug-report.yaml

```diff
@@ -0,0 +1,78 @@
+name: "\U0001F41B Bug report"
+description: Create a report to help Matter
+title: "[BUG] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting an issue against the Matter SDK! We need information about the bug report to follow up, so please help us out by filling out this information.
+  - type: textarea
+    id: repro
+    attributes:
+      label: Reproduction steps
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: bash
+    validations:
+      required: true
+  - type: input
+    id: prevalence
+    attributes:
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week, day, hour)"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: GitHub hash of the SDK that was being used
+      description: Hash of the GitHub SDK used
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Platform
+      description: What platforms are affected?
+      multiple: true
+      options:
+        - ameba
+        - android
+        - cc13x2_cc26x2
+        - darwin
+        - efr32
+        - esp32
+        - freeRTOS
+        - IMX8
+        - k32w
+        - nrf connect
+        - nrf
```

Comment:
are nrf and nrfconnect the same?

I would put linux as a separate item
Is vscode or windows needed?

Reply:
VSCode would be used to submit "VSCode bug" I assume? Removed nrfconnect